project_generation/content/templates/event/service/service.go.tmpl

```diff
@@ -35,6 +37,8 @@ func Run(ctx context.Context, serviceList *ExternalServiceList, buildTime, gitCo
 
 	// Get HTTP Server with collectionID checkHeader middleware
 	r := mux.NewRouter()
+	otelHandler := otelhttp.NewHandler(router, "/")
+	router.Use(otelmux.Middleware(cfg.OTServiceName))
 	s := serviceList.GetHTTPServer(cfg.BindAddr, r)
 
 	// Get Kafka consumer
```

Comment:
check with @gordonkerrONS here. We want to have his kafka changes in, so maybe this needs to be blocked until that's merged & released.